### DISCUSSION TITLE
Allow sorting to remove "The" from titles

### DISCUSSION THREAD

**Discussion by lukeskope** [Kareadita/Kavita#3037] [Ideas]
### Idea Description

When browsing series, would like a toggle to ignore "The" at the beginning of titles.  For example, sometimes Comic Vine tags Avengers comics as The Avengers, and other times simply Avengers.  I would like an option to ignore "The" in a series title so all Avengers titles appear together.  This is very common, CDisplayex has a toggle.  Plex ignores "The" by default, as does Music Bee.

### Idea Category

User Experience

### Duration of Using Kavita

2 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by DeepHueStudios** [comment]
I totally second this.  This functionality is pretty standard on most media management and server software. The ability to ignore "The" and "A" and all other language instances of this for example "El" "Les" etc.

**Reply by Nebelherr23** [reply]
I third this. While most of the reader apps account for this, I'd love for the Web UI to support this as well, so I have less of a reason to use a 3rd party app.

**Comment by Daniel9425** [comment]
I just came to the github page to ask for this exactly. I've been using Kavita for a few months now and I just noticed this the other day when I went browsing through my library. I think I didn't notice it before, but I typically search for what I want to read or am reading book listed under recently added.
I spent about 10 minutes going through all my books and modifying the Sort Name of any of my books that begin with "A", "An", or "The" to remove those articles from the sort name. But it would be better if Kavita could just do that automatically. For individual books that are not part of a series, the Sort Name that I set in Calibre would be the preferred source of the Sort Name in Kavita, but for Series names, I would like Kavita to remove/ignore articles at the beginning of the Series names.

**Comment by majora2007** [comment]
@scare376 Do you forsee this being needed for Japanese books? From what I'm reading, this doesn't seem common in Japanese/Chinese or other non-romanji languages (Hindi)

**Reply by scare376** [reply]
its not a thing in japanese as far as im aware.

**Comment by majora2007** [comment]
This is out on the nightly (v0.8.7.x) in case anyone wants to try it out and leave feedback. 

https://github.com/Kareadita/Kavita/pull/3895

**Reply by rmohns** [reply]
Hooray! Everything being grouped by “The” drives me crazy. :)

**Comment by LegendaryLass** [comment]
Asked for this years ago. Was told "Nah, we won't be doing that." Glad it's happening :)

**Reply by scare376** [reply]
This is why using the discussions is so important. Very few things in the app are 100% set in stone and these feature requests help to drive what changes need to happen. Most of the time that answer isn't it won't happen. It's that it can't happen right now. Joe is the primary dev for Kavita and has essentially been the only dev for the longest time. He has specific things that he wants to work on at a time and looks to these discussions for inspiration when it comes to changes that he can work on alongside some of the more major changes.
